docs/supported-commands.md

```diff
@@ -157,14 +157,15 @@
 
 ## Bit commands
 
-| Command  | Supported OR Not | Since Version | Desc |
-| -------- | ---------------- | ------------- | ---- |
-| GETBIT   | √                | v1.0.0        |      |
-| SETBIT   | √                | v1.0.0        |      |
-| BITCOUNT | √                | v1.0.0        |      |
-| BITPOS   | √                | v1.0.0        |      |
-| BITFIELD | √                | v2.7.0        |      |
-| BITOP    | √                | v2.1.0        |      |
+| Command     | Supported OR Not | Since Version | Desc |
+| ----------- | ---------------- | ------------- | ---- |
+| GETBIT      | √                | v1.0.0        |      |
+| SETBIT      | √                | v1.0.0        |      |
+| BITCOUNT    | √                | v1.0.0        |      |
+| BITPOS      | √                | v1.0.0        |      |
+| BITFIELD    | √                | v2.7.0        |      |
+| BITFIELD_RO | √                | -             |      |
```

Comment:
Hmmm should we change to v2.8.0 or just leave `-`? I've no idea

Reply:
I think we can use `unstable` or `v2.8.0`

Reply:
Guess we can change them at once after releasing 2.8.0

Reply:
I prefer unstable instead of -

Reply:
unstable is better